R/cf.R

```diff
@@ -146,16 +146,20 @@ jackknife_error <- function (samples, na.rm = FALSE) {
   sqrt(factor * (N - 1)^2 / N) * sd(samples)
 }
 
+
+#' Computes covariance matrix for jackknife samples. The rows containing any NA will be deleted if na.rm = TRUE.
+#' 
+#'
 #' @export
 jackknife_cov <- function (x, y = NULL, na.rm = FALSE, ...) {
     factor <- 1.0
     
     if (is.null(y)) {
         N <- nrow(x)
         if (na.rm) {
-            na_values <- apply(x, 2, function (row) any(is.na(row)))
+            na_values <- apply(x, 1, function (row) any(is.na(row)))
```

Comment:
We want to throw out the rows that contain `NA` values. Therefore margin 1 (rows) is the one to keep.